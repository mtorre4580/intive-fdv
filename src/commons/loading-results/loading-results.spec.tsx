import  React from 'react';
import renderer from 'react-test-renderer';
import LoadingResults from './loading-results';

describe('<LoadingResults />' , () => {
   
    it('should render snapshot', () => {
        const component = renderer.create(<LoadingResults />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

})
