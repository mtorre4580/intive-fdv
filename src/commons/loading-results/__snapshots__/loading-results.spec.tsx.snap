// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingResults /> should render snapshot 1`] = `
<div
  className="loader-results"
/>
`;
